Resources/Prototypes/Damage/modifier_sets.yml

```diff
@@ -106,8 +106,8 @@
     Slash: 1.2
     Piercing: 1.2
     Cold: 1.5
-    Poison: 0.8
-    Cellular: 0.2
+    Poison: 0.4
+    Cellular: 0.0
```

Comment:
this applies to slimepeople too right? does it need to be changed

Reply:
don't tell me that's the case

Reply:
fucking hell it is